build-w-docker.sh

```diff
@@ -7,6 +7,9 @@ set -e
 # reference: http://www.ostricher.com/2014/10/the-right-way-to-get-the-directory-of-a-bash-script/
 cd $(dirname $0)
 
+export MSYS_NO_PATHCONV=1
+export MSYS2_ARG_CONV_EXCL="*"
+
 docker run --rm \
     -v /`pwd`:/work \
```

Comment:
Since you added MSYS patch, you can remove the extra slash before `pwd`

```suggestion
    -v `pwd`:/work \
```